tests/util/test_file.cpp

```diff
@@ -95,9 +95,10 @@ SyncTestFile::SyncTestFile(const SyncConfig& sync_config)
     schema_mode = SchemaMode::Additive;
 }
 
-SyncTestFile::SyncTestFile(SyncServer& server)
+SyncTestFile::SyncTestFile(SyncServer& server, std::string name)
```

Comment:
So it is. I think GitHub must've decided to collapse the new file of tests when I did my find in page to check for where this was used. Sorry about that.